docs/java-api/client.asciidoc

```diff
@@ -167,7 +53,13 @@ Client client = TransportClient.builder().settings(settings).build();
 //Add transport addresses and do something with the client...
 --------------------------------------------------
 
-Or using `elasticsearch.yml` file as shown in <<node-client>>
+Or you can define `cluster.name` in the `/src/main/resources/elasticsearch.yml` file in your project.
+As long as `elasticsearch.yml` is present in the classpath, it will be used when you start your client.
```

Comment:
I don't think this is true? We do not look for elasticsearch.yml on the classpath. For the transport client, settings must be programmatically constructed and passed to the client builder.